plugins/woocommerce/src/Internal/Admin/WcPayWelcomePage.php

```diff
@@ -4,90 +4,57 @@
 
 use Automattic\WooCommerce\Admin\Features\OnboardingTasks\Tasks\WooCommercePayments;
 use Automattic\WooCommerce\Admin\WCAdminHelper;
+use Automattic\WooCommerce\Admin\PageController;
 
 /**
  * Class WCPayWelcomePage
  *
  * @package Automattic\WooCommerce\Admin\Features
  */
 class WcPayWelcomePage {
+	const TRANSIENT_NAME = 'wcpay_welcome_page_incentive';
+	const SCRIPT_NAME    = 'wcpay-welcome-page-incentive';
```

Comment:
I can't find any usage reference for this, do we need it?

Reply:
Great catch, I didn't remove it after going back to shared settings. Addressed in 36a69ff